slam_gmapping/launch/slam_gmapping.launch.py

```diff
@@ -3,7 +3,43 @@
 
 
 def generate_launch_description():
+    param_substitutions = {
+        'base_frame': 'base_link',
+        'odom_frame': 'odom',
+        'map_frame': 'map',
+        'map_update_interval': '3.0',
+        'maxUrange': '10.0',
+        'sigma': '0.05',
+        'kernelSize': '1',
+        'lstep': '0.05',
+        'astep': '0.05',
+        'iterations': '5',
+        'lsigma': '0.075',
+        'ogain': '3.0',
+        'lskip': '0',
+        'srr': '0.1',
+        'srt': '0.2',
+        'str': '0.1',
+        'stt': '0.2',
```

Comment:
Thanks for the addition of parameters. But this can be added as a separate PR since it is unrelated

Reply:
Hi @shiveshkhaitan.
I have removed these parameters in the launch file.